bus/bus.go

```diff
@@ -254,13 +255,25 @@ func (b *bus) bucketsHandlerGET(jc jape.Context) {
 }
 
 func (b *bus) bucketsHandlerPOST(jc jape.Context) {
-	var bucket api.Bucket
+	var bucket api.BucketCreateRequest
 	if jc.Decode(&bucket) != nil {
 		return
 	} else if bucket.Name == "" {
 		jc.Error(errors.New("no name provided"), http.StatusBadRequest)
 		return
-	} else if jc.Check("failed to create bucket", b.ms.CreateBucket(jc.Request.Context(), bucket.Name)) != nil {
+	} else if jc.Check("failed to create bucket", b.ms.CreateBucket(jc.Request.Context(), bucket.Name, bucket.Policy)) != nil {
+		return
+	}
+}
+
+func (b *bus) bucketsHandlerPolicyPUT(jc jape.Context) {
+	var req api.BucketUpdatePolicyRequest
+	if jc.Decode(&req) != nil {
+		return
+	} else if bucket := jc.PathParam("name"); bucket == "" {
+		jc.Error(errors.New("no bucket provided"), http.StatusBadRequest)
```

Comment:
`nit` maybe say "no name provided" bit misleading to not mention the missing parameter